src/commands/profile.ts

```diff
@@ -12,6 +15,35 @@ const __profile: InteractionHandlerPayloads.GuildChatInputCommand = {
         interaction.showModal(modal);
 
         // await results and collect
+        const submission = await interaction.awaitModalSubmit({
+            time: 240000,
+        });
+
+        // silly man did not do the submit in time like a mongrel
+        if (!submission) {
+            console.log(`Submission canceled due to timeout.`);
+            return;
+        }
+        submission.deferUpdate();
+
+        const name = submission.fields.getTextInputValue("name");
```

Comment:
For cleanliness, move into the try-catch block as it is being used only within that scope.